playbooks/ansible-galaxy-importer/run.yaml

```diff
@@ -10,7 +10,12 @@
       include_vars:
         file: '{{ _fetch.dest }}'
         name: galaxy_info
+
     - name: Confirm collection can be imported into galaxy
       args:
         chdir: "~/{{ zuul.projects['github.com/ansible-network/releases'].src_dir }}"
-      shell: "source .tox/venv/bin/activate; ./tools/validate-collection.sh ~/downloads/{{ galaxy_info.namespace}}-{{ galaxy_info.name }}-*.tar.gz"
+      shell: "source .tox/venv/bin/activate ; ./tools/validate-collection.sh ~/downloads/{{ galaxy_info.namespace}}-{{ galaxy_info.name }}-*.tar.gz"
+      environment:
+        GALAXY_IMPORTER_CONFIG: "/home/zuul/{{ zuul.project.src_dir }}/tests/galaxy-importer.cfg"
```

Comment:
Note: If the file is missing, galaxy importer **ignores** the missing file, this is safe even if collections don't include the config.